videoframes.py

```diff
@@ -146,7 +150,14 @@ def setup_actions(self):
 
             self.quality_actions.append(action)
 
-        self.context_menu.addMenu(quality_submenu)
+        # Non-empty lists are in a Python boolean context True
+        if self.quality_actions:
```

Comment:
If you check how the `self.quality_actions` is created, it's simply getting the quality options that was available when the stream **was opened**. If a stream has gone offline during the time of watching, this list will still contain the quality options from before and will therefore present the options and not the so desired Warning below. 

Do you get what I'm trying to say? :)